script.js

```diff
@@ -0,0 +1,69 @@
+const APIURL =
+    "https://api.themoviedb.org/3/discover/movie?sort_by=popularity.desc&api_key=04c35731a5ee918f014970082a0088b1&page=2";
+const IMGPATH = "https://image.tmdb.org/t/p/w1280";
+const SEARCHAPI =
+    "https://api.themoviedb.org/3/search/movie?&api_key=04c35731a5ee918f014970082a0088b1&query=";
+
+const main = document.getElementById("main");
+const form = document.getElementById("form");
+const search = document.getElementById("search");
+
+// Swiper initialization
+let swiper = new Swiper('.swiper-container', {
+    // Optional parameters
+    effect: 'coverflow', 
+    grabCursor: true,
+    centeredSlides: true,
+    slidesPerView: 'auto',
+    coverflowEffect: {
+        rotate: 50, 
+        stretch: 0, 
+        depth: 100, 
+        modifier: 1, 
+        slideShadows : true // 
+    },
+    /*pagination: {
+      el: '.swiper-pagination',
+      clickable: true,
+    
+    },*/
+});
+
+
+getMovies(APIURL);
+
+async function getMovies(url) {
+    const resp = await fetch(url);
+    const respData = await resp.json();
+    console.log(respData);
+    showMovies(respData.results);
+}
+
+function showMovies(movies) {
+    // Clear existing slides
+    swiper.removeAllSlides();
+

```

Comment:
use arrow functions

Reply:
@nabila39 I don't see anything related to the movies in the app, please recheck this